src/main/java/de/dhbw/karlsruhe/students/mailflow/core/domain/email/entities/Address.java

```diff
@@ -0,0 +1,36 @@
+package de.dhbw.karlsruhe.students.mailflow.core.domain.email.entities;
+
+import de.dhbw.karlsruhe.students.mailflow.core.domain.common.models.Entity;
+import de.dhbw.karlsruhe.students.mailflow.core.domain.email.valueObjects.AddressId;
+
+/**
+ * Representation of an e-mail address, consisting of a localPart and a domain,
+ * separated by an `@` sign.
+ */
+public class Address extends Entity<AddressId> {
+    private String localPart;
```

Comment:
nit: In other classes, you have an extra space at the beginning and end of a class. We should use one consistent code style. Are we able to enforce code styleguides here?